### ISSUE TITLE
emit doc/uri facts for C++ standard library names pointing to en.cppreference.com

### ISSUE THREAD

**Issue by shahms** [google/kythe#5374]
We create `doc` nodes and edges from comments present in code. These are almost universally lacking for C++ standard library implementations.

It should, however, be fairly straightforward to map from the standard library symbol definitions to the corresponding page on en.cppreference.com using the Doxygen tagfile present in the offline HTML book archive: https://en.cppreference.com/w/Cppreference:Archives

The doxygen tagfile format is, sadly, XML and I'm not particularly enthusiastic about growing an XML parser dependency.

**Comment by creachadair** [comment]
For simple purposes I've had good luck using https://pkg.go.dev/encoding/xml, which is in the standard library. It's not good for complex schemata, but probably fine for scraping doc links out of a ToC.

**Comment by shahms** [comment]
`tagfiles` are thankfully pretty simple, but the C++ indexer is C++ after all, but we could just use Go to translate the XML doc to protobuf or something

**Comment by creachadair** [comment]
> `tagfiles` are thankfully pretty simple, but the C++ indexer is C++ after all, but we could just use Go to translate the XML doc to protobuf or something

Yeah, I was thinking of a preprocessing step, since presumably it only needs to be updated occasionally.
